core/endorser/endorser.go

```diff
@@ -334,7 +336,9 @@ func (e *Endorser) ProcessProposal(ctx context.Context, signedProp *pb.SignedPro
 	return pResp, nil
 }
 
-func (e *Endorser) ProcessProposalSuccessfullyOrError(up *UnpackedProposal) (*pb.ProposalResponse, error) {
+// ProcessProposalSuccessfullyOrError processes the proposal and returns a response or error.
```

Comment:
There is an alternative, more (in my opinion) clean way of doing this for the endorser gRPC:

We already have several authentication filter which are essentially a chaining on endorsement functions. https://github.com/hyperledger/fabric/blob/f20eeb43733b4addd39ae4ced32d0faaca008fec/sampleconfig/core.yaml#L386 

To add a new built-in filter, we:
1. Put a construction of it in https://github.com/hyperledger/fabric/blob/master/core/handlers/library/library.go similarly to the existing functions
2. append it to the `handlers.authFilters` in core.yaml 

Or, alternatively we can always manually append if to the list at construction time like we do here:
https://github.com/hyperledger/fabric/blob/master/internal/peer/node/start.go#L835

This leaves the endorser code intact.

Reply:
Will not use the filter approach because it is better to throttle for as minimum code as needed. Otherwise, it may unnecessarily limit the concurrency.